workshops/drumpad/README.md

```diff
@@ -0,0 +1,470 @@
+---
+name: 'Drum Pad'
+description: 'Creating a Drum Pad in HTML, CSS & JS'
+author: '@emmanuel39hanks'
+---
+
+Ever wondered how you can play sounds with code? Well, if yes, then you will love this workshop! We will be creating a drum pad in less than 230 lines of code that plays actual sounds, and if you are a beginner, this won't only be fun and creative but will help you get comfortable with coding!
+
+<p align='center'>
+<img src="https://media.giphy.com/media/CjmvTCZf2U3p09Cn0h/giphy.gif" alt="ready-to-code" >
+</p>
+
+
+# Overview
+
+_Preview of the Drum Pad we are going to be creating_
+
+![Drum Pad Preview](https://cloud-ms6ubs6qu.vercel.app/0screencapture-drum-emmanuel39hanks-repl-co-2020-11-07-23_20_37.png)
+
+Apart from building the drum pad, you will also be learning about different types of events, functions, styling, and more with Vanilla JavaScript, HTML, and CSS. This whole workshop is customizable according to you, and the best part is, it will take less than 20 minutes to complete! 
+
+[Link to demo](https://drum.emmanuel39hanks.repl.co)
+
+## Getting started
+
+Let's start by setting up our IDE (Integrated Development Environment) on Rept.it, a free, online code editor.
```

Comment:
You mean repl.it, right? Add in a link too.

Reply:
Yes